test/Interop/Cxx/stdlib/avoid-import-cxx-math.swift

```diff
@@ -5,19 +5,17 @@
 import CxxStdlib
 
 func test() {
-    let x: Float = 1.0
+    let x: Double = 1.0
     let y: Double = 2.0
 
-    // Note: we dispatch `pow(Float,Double)`
-    // to ensure we don't pick up the
-    // C++ stdlib `pow` function template.
-    // The `pow` function is still reexported
-    // from Darwin via CxxStdlib, so there are
-    // matching overloads that can be found still.
-    // Note: the error is different on Glibc instead
-    // of Darwin, so do not check the exact error.
-    let _ =  CxxStdlib.pow(x, y) // expected-error {{}}
+    let _ =  pow(x, y)
 
-    let _ = CxxStdlib.abs(x) // expected-error {{module 'CxxStdlib' has no member named 'abs'}}
-    let _ = CxxStdlib.div(x) // expected-error {{module 'CxxStdlib' has no member named 'div'}}
+    let _ = abs(x)
+    // https://github.com/apple/swift/issues/67006
+    // let _ = div(42, 2)
```

Comment:
Perfect example of why we should test the positive case: div wasn't available on linux even before this patch because it's in `stdlib.h`. Issue filed.